.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm run lint
+      - run: npm run build
+      - run: npm run test
+
+  publish-npm:
```

Comment:
@ephread can you confirm that `test` and `publish-npm` are sequential and the second won't run if the first fails? This is what the doc seems to say but I had trouble finding a good explanation of this.

Reply:
Yes, steps are sequential. For instance, have a look at this [build log](https://github.com/ephread/Instructions/runs/658808761?check_suite_focus=true). The step named _Convert and upload coverage_ did not run because `Run tests` failed.